cni/network/invoker_cns.go

```diff
@@ -57,8 +58,15 @@ func (invoker *CNSIPAMInvoker) Add(nwCfg *cni.NetworkConfig, hostSubnetPrefix *n
 		return nil, nil, err
 	}
 
+	endpointId := GetEndpointID(args)
+	ipconfig := cns.IPConfigRequest{
+		OrchestratorContext: orchestratorContext,
+		PodInterfaceID:      endpointId,
+		InfraContainerID:    args.ContainerID,
+	}
+
 	log.Printf("Requesting IP for pod %v", podInfo)
-	response, err := invoker.cnsClient.RequestIPAddress(orchestratorContext)
+	response, err := invoker.cnsClient.RequestIPAddress(&ipconfig)
```

Comment:
log endpoint ID here as well

Reply:
updated log line on 68 to print both podInfo and ipconfig, which contains this information